### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Nw84**
I added some small fixes to the error messages i got when i tried to run build and i turned of the warning for using img instead of Image in eslintrc.json.

I also changed so that the latest movie added to the Movie database will show as the coming premier.

Lastly i added a response message if the request method was not POST in the login api

### CODE REVIEW

**Review by simonlj8 — APPROVED** [review]
Good! It looks much better with one movie image and that "Kommande Premiär" is above the picture.

**Review by mhema42 — APPROVED** [review]
I think everything looks fine when i looked att the branch/site, I had just a small suggestion for styling the image on mainhero.
Perfect that you fixed the errormessages and fixes to the imageproblem :)

Very good job as usual Tobias!

File: components/layout/MainHero.module.css [hunk 1]
```diff
@@ -18,9 +18,11 @@
     text-align: center;
     margin: 0 auto;
     font-size:2vw;
+    padding-bottom: 2.5rem;
 }
```
Comment (mhema42) [inline components/layout/MainHero.module.css:23]: I have a small suggestion to move the picture up a little bit and place a border around it...

here is my suggested styling code:

   padding-bottom: 1rem;                   _(this i changed from 2.5rem)_
}
.image {
    margin: 0 0 40px 0;                         _(added this line)_
    border: solid 3px rgb(107, 3, 3);      _(added this line)_
    border-radius: 3px;                         _(added this line)_
    max-height: 40vh;                           _(this i changed from 45vh)_

Now....
image

My suggestion...
image

Comment (Nw84) [reply components/layout/MainHero.module.css:23]: Good suggestions Mats, it makes the poster pop a bit more from the background 👍  I pushed the changes now

**Pull request merged**